File: app/ihm/src/domain/Menu.php
<?php

namespace domain;

class Menu
{
    protected $id;
    protected $nom;
    protected $createurId;
    protected $createurNom;
    protected $dateCreation;
    protected $dateMiseAJour;
    protected $plats;
    protected $prixTotal;

    public function __construct($id, $nom, $createurId, $createurNom, $dateCreation, $dateMiseAJour, $plats, $prixTotal)
    {
        $this->id = $id;
        $this->nom = $nom;
        $this->createurId = $createurId;
        $this->createurNom = $createurNom;
        $this->dateCreation = $dateCreation;
        $this->dateMiseAJour = $dateMiseAJour;
        $this->plats = $plats;
        $this->prixTotal = $prixTotal;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getNom()
    {
        return $this->nom;
    }

    public function getCreateurId()
    {
        return $this->createurId;
    }

    public function getCreateurNom()
    {
        return $this->createurNom;
    }

    public function getDateCreation()
    {
        return $this->dateCreation;
    }

    public function getDateMiseAJour()
    {
        return $this->dateMiseAJour;
    }

    public function getPlats()
    {
        return $this->plats;
    }

    public function getPrixTotal()
    {
        return $this->prixTotal;
    }
}

File: app/commandes/src/main/java/fr/univamu/iut/micromealscommandes/menu/Menu.java
package fr.univamu.iut.micromealscommandes;

public class Menu {

    protected int id;
    protected String nom;
    protected double prixTotal;

    public Menu() {
    }

    public Menu(int id, String nom, double prixTotal) {
        this.id = id;
        this.nom = nom;
        this.prixTotal = prixTotal;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public double getPrixTotal() {
        return prixTotal;
    }

    public void setPrixTotal(double prixTotal) {
        this.prixTotal = prixTotal;
    }

    @Override
    public String toString() {
        return "Menu{" + "id=" + id + ", nom='" + nom + '\'' + ", prixTotal=" + prixTotal + '}';
    }
}
